src/app/app-routing.module.ts

```diff
@@ -23,16 +23,20 @@ import { SearchTutorialComponent } from './modules/pages/search-tutorial/search-
 import { SuggestionComponent } from './modules/pages/suggestion/suggestion.component';
 import { SupportComponent } from './modules/pages/support/support.component';
 import { TechComponent } from './modules/pages/tech/tech.component';
+import { AboutComponent as AboutComponentEnUS } from './modules/pages/about-en_US/about.component';
+import { TechComponent as TechComponentEnUS } from './modules/pages/tech-en_US/tech.component';
 
 
 const routes: Routes = [
   { path: '', component: HomeComponent },
   { path: 'sobre', component: AboutComponent },
+  { path: 'sobre-en_US', component: AboutComponentEnUS },
```

Comment:
Funcionaria a gente usar a sigla do idioma antes do caminho da página?
```suggestion
  { path: 'en-US/sobre', component: AboutComponentEnUS },
```

Reply:
Funciona sim! Agora tá do jeito que indicasse